src/time/time.zig

```diff
@@ -551,7 +551,7 @@ pub const Duration = struct {
         return self.ns == other.ns;
     }
 
-    pub fn format(self: @This(), comptime _: []const u8, _: std.fmt.FormatOptions, writer: anytype) error{OutOfMemory}!void {
```

Comment:
Without this change, the compilation was failing with error

```
error: expected type 'error{NoSpaceLeft}', found 'error{OutOfMemory}'
```

This is because there is also the writer to fixed buffer that fails with `NoSpaceLeft`.

Having an error set that includes both `NoSpaceLeft` and `OutOfMemory`, ie `error{NoSpaceLeft, OutOfMemory}` did not work either.